src/dataExplorer/components/SchemaBrowserHeading.tsx

```diff
@@ -26,6 +26,7 @@ const SchemaBrowserHeading: FC = () => {
   const {selection} = useContext(PersistanceContext)
 
   const disableToggle: boolean = selection.composition?.diverged
+  const disableTooltipText = disableToggle ? FLUX_SYNC_DISABLE_TEXT : ''
```

Comment:
non-blocking request. right now we have sync on versus off, versus disabled. Not sure if future selves will get that disabled == diverged. Can we either can naming, or make a comment?

Thanks!